app/javascript/mastodon/features/status/components/detailed_status.js

```diff
@@ -17,6 +17,7 @@ import Icon from 'mastodon/components/icon';
 import AnimatedNumber from 'mastodon/components/animated_number';
 import PictureInPicturePlaceholder from 'mastodon/components/picture_in_picture_placeholder';
 import EditedTimestamp from 'mastodon/components/edited_timestamp';
+import StatusReactions from '../../../components/status_reactions';
```

Comment:
```suggestion
import StatusReactions from 'mastodon/components/status_reactions';
```
Vanilla imports are all over the place, but I personally prefer using the dirName as `../../../` seems a bit excessive.